release.json

```diff
@@ -13,6 +13,20 @@
         "JMXFETCH_VERSION": "0.38.0",
         "JMXFETCH_HASH": "f21fe4d604d56adb74423b940892178677cba635d6f1429c1606fc753c99aaed"
     },
+    "7.21.0-rc.1": {
+        "INTEGRATIONS_CORE_VERSION": "7.21.0-rc.1",
+        "OMNIBUS_SOFTWARE_VERSION": "7.21.0-rc.1",
+        "OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
```

Comment:
Why did the omnibus version scheme change from 7.20?

Reply:
Nice catch 🙇

Reply:
(took nightly entries)